Proxy/faraday_uart_stack/layer_4_service.py

```diff
@@ -129,7 +132,8 @@ def receive_datagram_queue_hasitem(self):
 
     def process_received_datagram(self, datagram):
         parsed_datagram_dict = layer_4_protocol.parse_packet(datagram)
-        #print "RX'd:", parsed_datagram_dict
+        if DEBUG:
```

Comment:
Added during unused variables commit. Was this caught by the flake8 tool @reillyeon ?
https://github.com/reillyeon/Faraday-Software/commit/0ca578306425529e416344f9d280714c62c59dec